Sprint-1/exercises/random.js

```diff
@@ -2,8 +2,8 @@ const minimum = 1;
 const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
-
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+5.66;
```

Comment:
The code at line 4 can generate a random number within a specific range. So what are the **possible** value of  `num`?

Reply:
I think each time you run the program, a different integer within the range [1, 100] will be generated.